site/content/en/latest/contributions/RELEASING.md

```diff
@@ -6,7 +6,10 @@ description: "This section tells the release process of Envoy Gateway."
 This document guides maintainers through the process of creating an Envoy Gateway release.
 
 - [Release Candidate](#release-candidate)
+  - [Prerequisites](#prerequisites)
```

Comment:
this is wrong.  my vs code is inserting these automatically on save.  will work on a commit to fix/revert this.